### ISSUE TITLE
Updates to package.json

### ISSUE THREAD

**Issue by seanmcbreen** [Microsoft/vscode-tools-for-ai#17]
Hi,

It would be great for you to include a link to your GH Issues in the package.json.  This will allow users to see a link for this in the marketplace and in VS Code.

```
"bugs": {
    "url": "https://github.com/Microsoft/vscode-wordcount/issues",
    "email": "smcbreen@microsoft.com"
},
```

![image](https://user-images.githubusercontent.com/1051190/33676224-00f0b922-da6a-11e7-8d9f-b3541225b620.png)

Additionally today you have no short description and that would improve display in marketplace search results.

```
"description": "Markdown Word Count Example - reports out the number of words in a Markdown file.",
```

![image](https://user-images.githubusercontent.com/1051190/33676198-e7a271ae-da69-11e7-9b85-c0fe1f2bfe57.png)

Sean
VS Code team

**Comment by chris-lauren** [comment]
Thanks for the tips Sean!

Do you know if there is similar approach to improve search results for our Visual Studio extension also? I have personally had issues finding it via search and suspect this is the reason

**Comment by seanmcbreen** [comment]
Yes the search relevance rating includes the extension title, description and tags.  So adding in the description should help a lot.

**Comment by qfyin** [comment]
Thanks for suggestion.
Will be included in 0.1.8 very soon.
